lib/ansible/plugins/callback/slack.py

```diff
@@ -90,6 +82,17 @@ def __init__(self, display=None):
                                   'installed. Disabling the Slack callback '
                                   'plugin.')
 
+        self.playbook_name = None
+
+        # This is a 6 character identifier provided with each message
+        # This makes it easier to correlate messages when there are more
+        # than 1 simultaneous playbooks running
+        self.guid = uuid.uuid4().hex[:6]
+
+    def set_options(self, options):
+
+        super(CallbackModule, self).set_options(options)
+
         self.webhook_url = self._plugin_options['webook_url']
```

Comment:
`webook_url` is still wrong here, shoudl be `webhook_url`

Reply:
dammit ... i forgot to fix the thing that triggered it all!